client/src/components/PublicConfirmationCard.js

```diff
@@ -32,37 +33,52 @@ const NavButton = styled.button`
   font-weight: bold;
   margin: 0px 5px 0px 5px;
```

Comment:
= as `margin: 0 5px` // 0 is for top and bottom, and 5px, right and left.

Reply:
Thx :D